specificatie/openapi.yaml

```diff
@@ -63,7 +63,25 @@ paths:
             type: integer
             minimum: 1
             maximum: 99999
-        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/parameters/fields"
+        - in: query
+          name: huisnummertoevoeging
+          description: Huisnummertoevoeging behorende bij een nummeraanduiding.
+          required: false
+          schema:
+            type: string
+            pattern: "^[0-9a-zA-Z]{1,4}$"
```

Comment:
Moet hier i.p.v. '{1,4}' in het pattern geen gebruik gemaakt worden van minLength en maxLength?
Zie https://github.com/VNG-Realisatie/Haal-Centraal-common/pull/112

Reply:
Na bespreking heb ik de maxLength en minLength verwijderd